types/templates.go

```diff
@@ -176,16 +180,35 @@ type IndexPageDataEpochs struct {
 	Ts                               time.Time     `json:"ts"`
 	Finalized                        bool          `json:"finalized"`
 	FinalizedFormatted               template.HTML `json:"finalized_formatted"`
-	EligibleEther                    uint64        `json:"eligibleether"`
-	EligibleEtherFormatted           template.HTML `json:"eligibleether_formatted"`
 	GlobalParticipationRate          float64       `json:"globalparticipationrate"`
 	GlobalParticipationRateFormatted template.HTML `json:"globalparticipationrate_formatted"`
 	VotedEther                       uint64        `json:"votedether"`
-	VotedEtherFormatted              template.HTML `json:"votedether_formatted"`
+	//VotedEtherFormatted              template.HTML `json:"votedether_formatted"`
+	ExecutionReward          *big.Int      `json:"-"`
+	ExecutionRewardFormatted template.HTML `json:"exec_reward"`
+	//EligibleEther                    uint64        `json:"eligibleether"`
+	//EligibleEtherFormatted           template.HTML `json:"eligibleether_formatted"`
 }
 
 // IndexPageDataBlocks is a struct to hold detail data for the main web page
 type IndexPageDataBlocks struct {
+	Epoch                    uint64        `json:"-"`
+	Slot                     uint64        `json:"slot"`
+	Ts                       time.Time     `json:"ts"`
+	Proposer                 uint64        `db:"proposer" json:"-"`
+	ProposerFormatted        template.HTML `json:"proposer_formatted"`
+	Status                   uint64        `db:"status" json:"-"`
+	StatusFormatted          template.HTML `json:"status_formatted"`
+	ProposerName             string        `db:"name" json:"-"`
+	ExecutionBlockNumber     int           `db:"exec_block_number" json:"exec_block_number"`
+	ExecutionReward          *big.Int      `json:"-"`
+	ExecutionRewardFormatted template.HTML `json:"exec_reward"`
+	ExecutionRewardRecipient template.HTML `json:"exec_reward_recipient"`
+	BlockRoot                []byte        `db:"blockroot" json:"-"`
+}
+
+// OldIndexPageDataBlocks is a struct to hold detail data for the old main web page, which for some reason is used in the validator and epoch handler
+type OldIndexPageDataBlocks struct {
```

Comment:
I feel like this has to be removed to avoid confusion. `BlocksPageDataBlocks` a few lines below looks like something that should be used instead, does it not?

Reply:
@LuccaBitfly , thoughts?

Reply:
Somehow overlooked this, sorry. You are probably right, but as both the validator and epoch page handler use the old block struct from the index page (which shouldn't have happened in the first place imo) and I don't see changing and testing these handlers within the scope of this task, I won't change this for now.

Reply:
As discussed, we have created a BIDS for this => Resolved